conf/core/ConfigOptions.asciidoc

```diff
@@ -3784,6 +3785,13 @@ The way subline matcher to use when determining matching sublines.
 Distance, in meters, used for sampling way data during angle histogram extraction with the
 SampledAngleHistogramExtractor
 
+=== way.joiner
```

Comment:
Since I made a lot of changes to the way joiner for Attribute Conflation and was wary about problems they would cause in other types of conflation, I separate them into two implementations for now.  After the new way joining is proven and tweaked some more, maybe we can combine them back to a single implementation.

Reply:
Sounds good.